archive/staticsource.go

```diff
@@ -5,22 +5,29 @@ import (
 
 	"github.com/brimsec/zq/driver"
 	"github.com/brimsec/zq/field"
+	"github.com/brimsec/zq/pkg/nano"
 	"github.com/brimsec/zq/zbuf"
 	"github.com/brimsec/zq/zng/resolver"
+	"github.com/brimsec/zq/zqe"
 )
 
-// staticSource is an implemetation of driver.MultiSource that provides
+// staticSource is an implementation of driver.MultiSource that provides
 // a single SpanInfo (with Chunks) to be processed by a zqd worker.
 // staticSource is used only for the zqd /worker call.
 type staticSource struct {
 	ark *Archive
-	si  SpanInfo
+	sis SpanInfoSource
```

Comment:
This will conflict with #1502, which does not use the archive and adds a driver.Source